scipy/odr/odrpack.py

```diff
@@ -171,9 +181,10 @@ class Data(object):
     Parameters
     ----------
     x : array_like
-        Input data for regression.
-    y : array_like, optional
-        Input data for regression.
+        Observed data for the independent variable of the regression
+    y : array_like or scalar, optional
+        If array_like, observed data for the dependent variable of the regression.
```

Comment:
This needs to have either double backticks or be spelled ``array-like``, because the underscore is reST markup